tests/common/devices/multi_asic.py

```diff
@@ -592,8 +598,8 @@ def docker_cmds_on_all_asics(self, cmd, container_name):
         """This function iterate for ALL asics and execute cmds"""
         duthost = self.sonichost
         if duthost.is_multi_asic:
```

Comment:
container name to be appended with asic index only for asic services.
if duthost.is_multi_asic and container_name in [DEFAULT_ASIC_SERVICE](https://github.com/Azure/sonic-mgmt/blob/2e10e91da0c12735427805c06b5e540a24ade79c/tests/common/devices/sonic.py#L35)